lib/virt_disk/block_file.rb

```diff
@@ -1,9 +1,10 @@
 module VirtDisk
   class BlockFile
-    attr_accessor :path
+    attr_accessor :path, :offset
 
-    def initialize(path)
+    def initialize(path, offset=0)
```

Comment:
@movitto I would resist having anything depend on the `virt_disk` gems until we have the basic functionality in place in it's desired form.

Given the `BlockIO` class currently in `VirtFS` is already a temporary class (it will be going away when dirt_disk is available) I would prefer the offset workaround you need be implemented there.